.github/workflows/karma.yml

```diff
@@ -135,6 +135,7 @@ jobs:
             - run: API_VERSION=61 yarn sauce:ci
             - run: API_VERSION=61 DISABLE_SYNTHETIC=1 yarn sauce:ci
             - run: DISABLE_SYNTHETIC_SHADOW_SUPPORT_IN_COMPILER=1 DISABLE_SYNTHETIC=1 yarn sauce:ci
+            - run: DISABLE_SYNTHETIC_SHADOW_SUPPORT_IN_COMPILER=1 DISABLE_SYNTHETIC=1 DISABLE_STATIC_CONTENT_OPTIMIZATION=1 yarn sauce:ci
```

Comment:
Just another case I think is worth testing, especially if we start actually shipping `disableSyntheticShadowSupport=true`.

Reply:
Eagerly anticipating another `chore(ci): rebalance karma tests`

Reply:
You guessed it! It's imbalanced again. I wish GH had an easier way to do this. 🫠